doc/source/ray-overview/getting-started.md

```diff
@@ -427,11 +380,11 @@ Learn more about Ray RLlib
 
 `````
 
-## Ray Core Quick Start
+## Ray Core Quickstart
 
 Ray Core provides simple primitives for building and running distributed applications.
-Below you find examples that show you how to turn your functions and classes easily into Ray tasks and actors,
-for both Python and Java.
+Below are examples of how to turn functions and classes easily into Ray tasks and actors,
+for Python and Java.
```

Comment:
We should include Java installation instructions here.

Reply:
ack